.github/DISCUSSION_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,51 @@
+title: "[Bug] "
+labels: ["Q&A"]
+body:
+  - type: markdown
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      label: Steps to reproduce.
+      value: |
+        1. Go to '...'
+        2. Click on '....'
+        3. Scroll down to '....'
+      ...
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      label: Expected behavior
+      description: A clear and concise description of what you expected to happen.
+  - type: dropdown
+    attributes:
+      label: Python Version
+      options:
+        - 3.7
+        - 3.8
+        - 3.9
+        - 3.10
+        - 3.11
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Django Version
+      options:
+        - 2.2
+        - 3.0
+        - 3.1
+        - 3.2
+        - 4.0
+        - 4.1
+        - 4.2
```

Comment:
Same as with Python version field, I don't think we should require this detail of maintenance